db/structure.sql

```diff
@@ -155,11 +182,62 @@ CREATE TABLE public.children (
 
 
 --
--- Name: data_migrations; Type: TABLE; Schema: public; Owner: -
+-- Name: lookup_cities; Type: TABLE; Schema: public; Owner: -
 --
 
-CREATE TABLE public.data_migrations (
-    version character varying NOT NULL
```

Comment:
I'm a little unclear why the data migrations table would come out of the schema with these changes, can you investigate?

Reply:
Aha.  I didn't realize we needed to be using `db:migrate:with_data`  I was missing the **with_data** part.

In another project I've worked on, we've written a task that calls all of the steps to:
- drop the db (if they exist)
- create the db
- db:migrate
- seed

So there's just 1 task to always run in development.  
We could write this task and then be sure it uses `db:migrate:with_data`
(as a separate PR).

Reply:
Feel free to add that task if you like!